src/storage/src/monitor/hummock_state_store_metrics.rs

```diff
@@ -167,15 +163,15 @@ impl HummockStateStoreMetrics {
         let opts = histogram_opts!(
                 "state_store_write_batch_duration",
                 "Total time of batched write that have been issued to state store. With shared buffer on, this is the latency writing to the shared buffer",
-                exponential_buckets(0.0001, 2.0, 21).unwrap() // max 104s
+                exponential_buckets(0.0001, 2.0, 21).unwrap() // min 1ms ~ max 104s
             );
         let write_batch_duration =
             register_histogram_vec_with_registry!(opts, &["table_id"], registry).unwrap();
 
         let opts = histogram_opts!(
             "state_store_write_batch_size",
             "Total size of batched write that have been issued to state store",
-            exponential_buckets(10.0, 2.0, 25).unwrap() // max 160MB
+            exponential_buckets(256.0, 2.0, 25).unwrap() // min 256B ~ max 4GB
```

Comment:
Seems incorrect. The range is  256B~2GB.
The buckets is 
```
[256.0, 512.0, 1024.0, 2048.0, 4096.0, 8192.0, 16384.0, 32768.0, 65536.0, 131072.0, 262144.0, 524288.0, 1048576.0, 2097152.0, 4194304.0, 8388608.0, 16777216.0, 33554432.0, 67108864.0, 134217728.0, 268435456.0, 536870912.0, 1073741824.0, 2147483648.0, +Inf]
```

Reply:
BTW, since we only measure p_max of this histogram, if the mem_table size is larger than 2GB(for example 4GB, 6GB), it will show 2GB(the upper bound value of the last bucket).

Reply:
> Seems incorrect. The range is 256B~2GB. The buckets is
> 
> ```
> [256.0, 512.0, 1024.0, 2048.0, 4096.0, 8192.0, 16384.0, 32768.0, 65536.0, 131072.0, 262144.0, 524288.0, 1048576.0, 2097152.0, 4194304.0, 8388608.0, 16777216.0, 33554432.0, 67108864.0, 134217728.0, 268435456.0, 536870912.0, 1073741824.0, 2147483648.0, +Inf]
> ```

Nope, the last `+inf` bucket is not counted in "25" according to the [doc](https://docs.rs/prometheus/latest/prometheus/fn.exponential_buckets.html):

> The final +Inf bucket is not counted and not included in the returned slice

So there will be actually 26 buckets: `[256.0, 512.0, ... , 2147483648.0, 4294967296.0, +inf]`

Reply:
> BTW, since we only measure p_max of this histogram, if the mem_table size is larger than 2GB(for example 4GB, 6GB), it will show 2GB(the upper bound value of the last bucket).

Correct. This is a known limitation.

If you are interested, may take a look on this discussion ☺️ https://twitter.com/fuyufjh/status/1687095309594415104 It's possible to record an accurate max, but not very straight-forward, so let's keep it simple & stupid now.

Reply:
Agree.